Mathlib/Topology/Algebra/Module/ModuleTopology.lean

```diff
@@ -413,4 +413,84 @@ lemma _root_.ModuleTopology.eq_coinduced_of_surjective
 
 end surjection
 
+section Prod
+
+variable {R : Type*} [TopologicalSpace R] [Semiring R]
+variable {M : Type*} [AddCommMonoid M] [Module R M] [TopologicalSpace M] [hM : IsModuleTopology R M]
+variable {N : Type*} [AddCommMonoid N] [Module R N] [TopologicalSpace N] [hN : IsModuleTopology R N]
+
+/-- The product of the module topologies for two modules over a topological ring
+is the module topology. -/
+instance prod : IsModuleTopology R (M × N) := by
+  constructor
+  haveI : ContinuousAdd M := toContinuousAdd R M
+  haveI : ContinuousAdd N := toContinuousAdd R N
```

Comment:
```suggestion
  have : ContinuousAdd M := toContinuousAdd R M
  have : ContinuousAdd N := toContinuousAdd R N
```

Reply:
Do you understand the difference? There is no discernable difference in the infoview *shrug*. All I know is "I means inline now".